test/scripts/e2e_client_runner.py

```diff
@@ -471,6 +471,14 @@ def main():
     goal_network_stop(netdir, env, normal_cleanup=True)
     if not args.keep_temps:
         goal_network_delete(netdir, normal_cleanup=True)
+    else:
```

Comment:
Now that there's an `else`, I'd rather swap the predicate so it's not testing `not`.